input/pagecontent/processing_model.md

```diff
@@ -0,0 +1,136 @@
+The following description provides an algorithm for how to process a FHIR
+resource as input for a `ViewDefinition`. Implementations do not need to follow
+this algorithm directly, but their outputs should be consistent with what this
+model produces.
+
+### Validate Columns (entry point)
+
+**Purpose**: This step ensures that a ViewDefinition's columns are valid, by setting up a recursive call.
+
+**Inputs**
+* `V`: a `ViewDefinition` to validate
+
+1. Call `ValidateColumns([], V)` according to the recursive step below.

```

Comment:
Are the ValidateColumns arguments in the correct order here?

Reply:
Nope, flipped! Thanks, will fix.